invadedhouse/models/house.py

```diff
@@ -0,0 +1,190 @@
+from .chest import Chest
+from .item import Item
+from .monster import Monster
+from .room import Room
+from .weapon import Weapon
+
+
+class House:
+    def __init__(self):
+        self.starting_location = None
+        self.room_directory = None
+        self.build_house()
+
+    def build_house(self):
+        # set up the rooms
```

Comment:
Maybe now you're beginning to see... the code in this `build_house` function is very specific to our Invaded House game... but what if we created functions inside the `House` class that let you set up the house (or map) from some other code? Then all of a sudden, this class could be re-used in both your game and Jake's Dinosaur game... 🤔

Reply:
YEAH!!!!!!!!!!!!! YEAH!!!!!!!!!! I could share the code or something